nix/marlowe-playground/easy-purescript-nix.nix

```diff
@@ -0,0 +1,6 @@
+{ inputs, pkgs, system, ... }:
+let
+  easy-purescript-nix = pkgs.callPackage inputs.iogx.inputs.easy-purescript-nix { };
```

Comment:
If we have this as an upstream input why not just automatically add it to `pkgs` as an overlay?